.ci/release_tests/example_2.py

```diff
@@ -1,8 +1,12 @@
+import torch
 from torch.utils.data import DataLoader
 from torchvision import datasets, transforms
 
+# adaptive_avg_pool2d_backward_cuda in mnist_classifier is not deterministic
+torch.use_deterministic_algorithms(False)
```

Comment:
this has to be included in README snippet since `mnist_model` uses `adaptive_avg_pool2d_backward_cuda` from `F.adaptive_avg_pool2d`. It's a little ugly sad

Reply:
CC: @ravi-mosaicml lmk if u object to this (it's also in README). I don't love hacky code going out in READMEs... but I think it's not a big deal. If you're good w it, resolve and PR will merge :)

Reply:
Could also remove adaptive pool from mnist... let me see if thats an option

Reply:
Hmm, would prefer not to update the readme to include this line. Could we modify the `test_release_tests_reflect_readme` function to allow for a header or extra lines that can appear in this file?